services/DatastoreService.go

```diff
@@ -0,0 +1,165 @@
+package services
+
+import (
+	"context"
+	"github.com/onego-project/onego/blueprint"
+	"github.com/onego-project/onego/requests"
+	"github.com/onego-project/onego/resources"
+)
+
+// DatastoreService structure to manage OpenNebula datastore
+type DatastoreService struct {
+	Service
+}
+
+const enable = true
```

Comment:
You can ommit this and use `true` and `false` instead. It's a bit confusing especially when there is a method called `enable` with parameter `enable`  :see_no_evil: